.dockerignore

```diff
@@ -0,0 +1,9 @@
+# Ignore everything
+**
+
+# Allow files and directories
+!/client
```

Comment:
All client code is replaced by `/dist`

Reply:
is generated into `dist/`*

Reply:
This is no longer the case, the client folder can be ignored now and removed out of this file

Reply:
Fixed.